pkg/services/vmoperator/vmopmachine_test.go

```diff
@@ -70,6 +70,7 @@ var _ = Describe("VirtualMachine tests", func() {
 		vmIP                     = "127.0.0.1"
 		biosUUID                 = "test-biosUuid"
 		missingK8SVersionFailure = "missing kubernetes version"
+		ClusterNameLabel         = "cluster.x-k8s.io/cluster-name"
```

Comment:
Let's reuse clusterv1.ClusterNameLabel instead of redefining it here

Reply:
Sure, fixed it